utils/formatURLInput.js

```diff
@@ -0,0 +1,9 @@
+// check to see if url string begins with 'http' and if not, add 'http://' to beginning of string
+const formatURLInput = (url) => {
```

Comment:
Food for thought - future function that can handle https.

Reply:
I think adding just the HTTP protocol is the way to go because websites will redirect to HTTPS for you, but I don't believe they will redirect from HTTPS to HTTP in the rare case that a website does not use HTTPS.